src/libraries/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -323,6 +323,144 @@ public enum TYMED
         TYMED_ENHMF = 64,
     }
 }
+namespace System.Runtime.InteropServices.Marshalling
+{
+    [System.AttributeUsageAttribute(System.AttributeTargets.Class, Inherited = false)]
+    [System.CLSCompliantAttribute(false)]
+    public sealed partial class ComExposedClassAttribute<T> : System.Attribute, System.Runtime.InteropServices.Marshalling.IComExposedDetails where T : System.Runtime.InteropServices.Marshalling.IComExposedClass
+    {
+        public ComExposedClassAttribute() { }
+        public unsafe System.Runtime.InteropServices.ComWrappers.ComInterfaceEntry* GetComInterfaceEntries(out int count) { throw null; }
+    }
+    public sealed partial class ComObject : System.Runtime.InteropServices.IDynamicInterfaceCastable, System.Runtime.InteropServices.Marshalling.IUnmanagedVirtualMethodTableProvider
+    {
+        internal ComObject() { }
+        public void FinalRelease() { }
```

Comment:
```suggestion
        public void FinalRelease() { throw null; }
```

Reply:
We only `throw null;` in a ref assembly when the return is non-void.